README.md

```diff
@@ -63,6 +63,7 @@ alternatives tend to be poorly-received**.
 * Submit a pull request. As a pull request the RFC will receive design
 feedback from the larger community, and the author should be prepared
 to revise it in response.
+* Find a champion on the ember-cli core team. Without a champion, the RFC will not progress.
```

Comment:
> What's the motivation behind this?

Today, this is implicit I would like to make it explicit. Nothing lands unless someone on the core-team is able to represent the RFCs interests at the meetings. Today this ends up happening quite ad-hoc, by formalizing this I hope to empower more successful RFCs.

> Also this requires to have a list/link to who are the current people on ember-cli core team. Someone relatively new to Ember world won't know who all are on the core team and who he should contact for which area.

You are correct, it should link to "who to get a champion" which should include who and how.